Makefile

```diff
@@ -22,29 +21,51 @@ DEFAULT_IMAGES = ddev-php-base ddev-php-prod
 VERSION := $(shell git describe --tags --always --dirty)
 BUILDINFO = $(shell echo hash=$$(git rev-parse --short HEAD) Built $$(date) by $${USER} on $$(hostname) $(BUILD_IMAGE) )
 
+# In CI environments, use the plain Docker build progress to not overload the CI logs
+PROGRESS := $(if $(CI),plain,auto)
+
 #
 # This version-strategy uses a manual value to set the version string
 #VERSION := 1.2.3
 
-DOCKER_BUILDKIT=1
-
 build: images
 
 images: $(DEFAULT_IMAGES)
 
-push: images
-	for item in $(DEFAULT_IMAGES); do docker push $(DOCKER_ORG)/$$item:$(VERSION); echo "pushed $(DOCKER_ORG)/$$item"; done
+$(DEFAULT_IMAGES): prep .docker-build-info.txt
+	set -eu -o pipefail; \
+	DOCKER_BUILDKIT=1 docker buildx build --progress=$(PROGRESS) $(BUILDPUSHARG) --platform linux/amd64 --label com.ddev.buildhost=${shell hostname} --target=$@  -t $(DOCKER_ORG)/$@:$(VERSION) $(DOCKER_ARGS) .
```

Comment:
DOCKER_BUILDKIT=1 is no longer needed with Docker buildx, as [it uses Buildkit by default](https://docs.docker.com/engine/reference/commandline/buildx/). Only found out about this yesterday 😊

Reply:
I knew they were moving that way, but it seems to me the move might not be done yet. They had a bug in macOS with default buildkit on, and I thought they backed away.

https://docs.docker.com/develop/develop-images/build_enhancements/ still shows it as normal to turn it on with DOCKER_BUILDKITE=1, but I'm sure we're going to be wanting to remove that.